WISHLIST.txt

```diff
@@ -14,4 +14,21 @@
 - craftable, upgradeable sentry bot. floats behind the player and shoots stuff
     - better batteries increase HP? or shields?
     - switchable weapon
-    - inventory/storage?
\ No newline at end of file
+    - inventory/storage?
+    
+    *AugiteSoul's ideas, edit them whenever you want ^^*
+- computercraft compat w/ MS peripherals, and auto-transit using those
+- tractor beams, forcingly move a ship, uses lots of RF
+- wide controllers, those can control machines you own in a large radius(disable all air locks?)

```

Comment:
Um, you're right. This would be only useful in very large MS, and - you could still use redstone and a lever for this.